content/acra/configuring-maintaining/general-configuration/acra-rollback.md

```diff
@@ -77,6 +77,217 @@ Zones are deprecated since 0.94.0, will be removed in 0.95.0.
   File for store inserts queries. 
   Default is `decrypted.sql`.
 
+#### TLS
+
+* `--tls_database_enabled=<true|false>`
+
+  Turns on/off TLS for connection with Database to `--db_connection_string`.
+
+  * `true` — turns on
+  * `false` — (default) turns off.
+
+* `--tls_auth=<mode>`
+
+  Set authentication mode that will be used for TLS connection.
+
+  * `0` — do not request client certificate, ignore it if received;
+  * `1` — request client certificate, but don't require it;
+  * `2` — expect to receive at least one certificate to continue the handshake;
+  * `3` — don't require client certificate, but validate it if client actually sent it;
+  * `4` — (default) request and validate client certificate.
+
+  These values correspond to [crypto.tls.ClientAuthType](https://golang.org/pkg/crypto/tls/#ClientAuthType).
+
+* `--tls_key=<filename>`
+
+  Path to private key that will be used for TLS handshake.
+  Should correspond to the certificate configured with `--tls_cert`.
+  Empty by default.
+
+* `--tls_cert=<filename>`
+
+  Path to TLS certificate that will be sent to other peers during handshake.
+  It will be sent to the client application (or AcraConnector if it's used between client app and AcraServer) and to the
```

Comment:
It will be sent to the database, not an application or connector. Take an example from [acra-keys](https://docs.cossacklabs.com/acra/configuring-maintaining/general-configuration/acra-keys/list/#hashicorp-vault). Or much better from other PR with fixed descriptions (mentioning AcraServer) - https://github.com/cossacklabs/product-docs/pull/291
But change Vault to database.